velox/docs/functions/spark/datetime.rst

```diff
@@ -187,6 +187,13 @@ These functions support TIMESTAMP and DATE input types.
     Returns the `ISO-Week`_ of the year from x. The value ranges from ``1`` to ``53``.
     A week is considered to start on a Monday and week 1 is the first week with >3 days.
 
+.. function:: weekday(date/timestamp) -> integer
```

Comment:
Please remove timestamp. Ditto for `dayofweek`.

Reply:
Done, thanks!